x-pack/solutions/security/plugins/security_solution/public/cloud_security_posture/components/csp_details/misconfiguration_findings_details_table.tsx

```diff
@@ -95,9 +95,17 @@ export const MisconfigurationFindingsDetailsTable = memo(
 
     const [currentFilter, setCurrentFilter] = useState<string>('');
 
+    const [sortField, setSortField] = useState<
+      MISCONFIGURATION.RESULT_EVALUATION | MISCONFIGURATION.RULE_NAME | 'resource' | 'rule'
```

Comment:
Perhaps moving the types to  specific type   `SortFieldType`. This will be easier to read  and understand the context of the useState values.